pymc3/distributions/posterior_predictive.py

```diff
@@ -75,24 +70,24 @@ class _TraceDict(_TraceDictParent):
 
     varnames: List[str]
     _len: int
-    data: Dict[str, np.ndarray]
+    data: Point
 
     def __init__(
         self,
-        point_list: Optional[List[Dict[str, np.ndarray]]] = None,
+        point_list: Optional[List[Point]] = None,
         multi_trace: Optional[MultiTrace] = None,
-        dict: Optional[Dict[str, np.ndarray]] = None,
+        dict_: Optional[Point] = None,
     ):
         """"""
         if multi_trace:
-            assert point_list is None and dict is None
-            self.data = {}  # Dict[str, np.ndarray]
```

Comment:
this is already annotated above (`data: Point`), no need to re-annotate it here. `reveal_type` shows it's picked up correctly anyway